modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -40,4 +45,8 @@ If Flow Designer was unable to publish the template, check that your user ID has
 
 == What to do next
 
-On the template's page in Exchange, provide a description, screeshots, or videos to explain the purpose of the template and how to use it.
+On the template's page in Exchange, provide a description, screenshots, or videos to explain the purpose of the template and how to use it.
+
+Whenever you need to update the template, update your Mule application and click *Publish*. The version number increments automatically.
+
+Whenever you want to update the information that is in Exchange about the template, edit the template's page in Exchange. For example, you cannot change the name of the template when you publish updates from Flow Designer. However, you can change the name in Exchange.
```

Comment:
I think this is better: "Whenever you want to update the template information in Exchange,"